gpio-adapter.js

```diff
@@ -61,7 +87,10 @@ class GpioDevice extends Device {
     }
     if (pinConfig.direction == 'in') {
       if (pinConfig.edge === undefined) {
-        pinConfig.edge = 'none';
+        pinConfig.edge = 'both';
+      }
+      if (pinConfig.debounce === undefined) {
```

Comment:
Direct comparison to `undefined` is unsafe, as it can be re-defined. Rather, you should do one of the following:
`typeof pinConfig.debounce === 'undefined'`
`pinConfig.hasOwnProperty('debounce')`